VideoWeb/VideoWeb/ClientApp/src/app/vh-officer/participant-info-tooltip/participant-info-tooltip.component.ts

```diff
@@ -8,5 +10,14 @@ import { ParticipantContactDetails } from '../../shared/models/participant-conta
 })
 export class ParticipantInfoTooltipComponent {
     @Input() participant: ParticipantContactDetails;
-    constructor() {}
+    constructor(private translateService: TranslateService) {}
+    getHearingRole() {
+        let hearingRole = this.participant.hearingRole;
+        if (this.participant.role === Role.QuickLinkParticipant) {
+            hearingRole = this.translateService.instant('participant-info-tooltip.quick-link-participant');
+        } else if (this.participant.role === Role.QuickLinkObserver) {
+            hearingRole = this.translateService.instant('participant-info-tooltip.quick-link-observer');
+        }
```

Comment:
I was thinking we should use translate pipe but none of the other roles have translations on this. Do we need Welsh support for this?

Reply:
I can query Lorenzo about it, if he doesnt think translation is required do you want me to remove the use of translate service here?

Reply:
Yeah. I guess the same could go for the copy link stuff which I know there's a story for Welsh but if we don't need it we shouldn't take the time for it.

Reply:
Lorenzo confirmed we dont need welsh translation here, so I will be removing all reference to translate service and the translate pipe. I will store the display text as string const inside of this component as discussed with Farron